test/manual/amp-lightbox-gallery-launch.amp.html

```diff
@@ -123,14 +149,56 @@ <h1>Lightbox Carousel Test</h1>
       layout="fill"
       />
     </amp-facebook>
-    <amp-ad width="300"
+  </amp-carousel>
+
+<h1>Lightbox Carousel Ad Support Test</h1>
+  <amp-carousel height="300"
+    width="400"
+    layout="fixed"
+    type="slides"
+    lightbox>
+    <amp-img src="https://images.unsplash.com/photo-1503843778847-2b8bdce2ed3d"
+      width="400"
+      height="300"
+      alt="a beautiful cat"></amp-img>
+      <amp-ad width="300"
       height="250"
       type="a9"
       data-aax_size="300x250"
       data-aax_pubname="test123"
       data-aax_src="302">
     </amp-ad>
-
+    <amp-ad
+      layout="responsive"
+      width="350"
+      height="250"
+      id="i-amphtml-demo-1"
```

Comment:
Is this necessary for ads purposes or test purposes? Is there a way to test this without using `id`?

Reply:
see comment above